CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ CHANGELOG
 
 ## HEAD (Unreleased)
 
+- [CLI] Add pagination options to `pulumi stack history` (`--page`, `--pageSize`). These replace the `--limit` flag.
```

Comment:
Should it be `page-size` instead of `pageSize`? I looked at other flags in pulumi and it [seems](https://github.com/pulumi/pulumi/blob/3236c9ad3fa44fdfdc8a3fddf7b9621bd48e80ac/pkg/cmd/pulumi/refresh.go#L166) [like](https://github.com/pulumi/pulumi/blob/3236c9ad3fa44fdfdc8a3fddf7b9621bd48e80ac/pkg/cmd/pulumi/up.go#L421-L425) `hyphen-case` is common, and I didn't see examples of camelCase but I didn't look very hard.

Reply:
Agreed it should be hyphen based flags

Reply:
👍  good catch. I pushed the change up. take another look.